Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1665,10 +1657,11 @@ def prepare_RECOBEFMIX(self, sequence = "reconstruction"):
 
     def prepare_PAT(self, sequence = "miniAOD"):
         ''' Enrich the schedule with PAT '''
-	self.prepare_PATFILTER(self)
-        self.loadDefaultOrSpecifiedCFF(sequence,self.PATDefaultCFF,1) #this is unscheduled
-	if not self._options.runUnscheduled:	
-		raise Exception("MiniAOD production can only run in unscheduled mode, please run cmsDriver with --runUnscheduled")
+        self.prepare_PATFILTER(self)
+        self.loadDefaultOrSpecifiedCFF(sequence,self.PATDefaultCFF)
+        self.scheduleSequence(sequence.split('.')[-1], 'pat_step')
+        if not self._options.runUnscheduled:
```

Comment:
If the original use of `runUnscheduled` also ran the `convertToUnscheduled` then I am fine with the present use.